build/deploy.js

```diff
@@ -100,10 +102,15 @@ class Deploy {
   }
 
   async init() {
-    this.namespace = process.env.WSK_NAMESPACE || 'helix-index';
+    const wskPropsFile = process.env.WSK_CONFIG_FILE || path.resolve(os.homedir(), '.wskprops');
+    let wskProps = {};
+    if (await fse.pathExists(wskPropsFile)) {
+      wskProps = dotenv.parse(await fse.readFile(wskPropsFile));
+    }
+    this.namespace = process.env.WSK_NAMESPACE || wskProps.NAMESPACE || 'helix-index';
```

Comment:
Not only takes `WSK_CONFIG_FILE` variable but also a `.wskprops` file into account: not sure, whether this could introduce some new behavior that might be confusing?